### ISSUE TITLE
Blind XXE lesson: The page http://localhost:8081/ping doesn't exist

### ISSUE THREAD

**Issue by h-rueda** [WebGoat/WebGoat#456]
Chapter: XXE
Page: 6
Lesson: Blind XXE
URL: http://localhost:8080/WebGoat/start.mvc#lesson/XXE.lesson/5

That exercise asks us to ping the WebWolf server with the following HTTP request:
`http://localhost:8081/ping?text=HelloWorld`
But the `/ping` page doesn't exist in WebWolf.

The response I get is

```
Whitelabel Error Page

This application has no explicit mapping for /error, so you are seeing this as a fallback.
Sun Apr 29 17:31:44 CEST 2018
There was an unexpected error (type=Not Found, status=404).
No message available
```

**Comment by nbaars** [comment]
Correct, updated the links to WebWolf by introducing an asciidoc macro which takes care of the correct replacement of the WebWolf host depending on running it local or in container

Thanks for reporting

**Comment by nbaars** [comment]
Closing, as we released v8.0.0.M15
